sdc/hiframes/pd_series_ext.py

```diff
@@ -1032,72 +1032,72 @@ def generic_expand_cumulative_series(self, args, kws):
         if attr in SeriesAttribute.__dict__:
             delattr(SeriesAttribute, attr)
 
-# PR135. This needs to be commented out
-@infer_global(operator.getitem)
-class GetItemSeries(AbstractTemplate):
-    key = operator.getitem
-
-    def generic(self, args, kws):
-        assert not kws
-        [in_arr, in_idx] = args
-        is_arr_series = False
-        is_idx_series = False
-        is_arr_dt_index = False
-
-        if not isinstance(in_arr, SeriesType) and not isinstance(in_idx, SeriesType):
-            return None
-
-        if isinstance(in_arr, SeriesType):
-            in_arr = series_to_array_type(in_arr)
-            is_arr_series = True
-            if in_arr.dtype == types.NPDatetime('ns'):
-                is_arr_dt_index = True
-
-        if isinstance(in_idx, SeriesType):
-            in_idx = series_to_array_type(in_idx)
-            is_idx_series = True
-
-        # TODO: dt_index
-        if in_arr == string_array_type:
-            # XXX fails due in overload
-            # compile_internal version results in symbol not found!
-            # sig = self.context.resolve_function_type(
-            #     operator.getitem, (in_arr, in_idx), kws)
-            # HACK to get avoid issues for now
-            if isinstance(in_idx, (types.Integer, types.IntegerLiteral)):
-                sig = string_type(in_arr, in_idx)
-            else:
-                sig = GetItemStringArray.generic(self, (in_arr, in_idx), kws)
-        elif in_arr == list_string_array_type:
-            # TODO: split view
-            # mimic array indexing for list
-            if (isinstance(in_idx, types.Array) and in_idx.ndim == 1
-                    and isinstance(
-                        in_idx.dtype, (types.Integer, types.Boolean))):
-                sig = signature(in_arr, in_arr, in_idx)
+if sdc.config.config_pipeline_hpat_default:
```

Comment:
How this change is related to `describe` operator implementation?

Reply:
Describe for string series calls iloc, which calls getitem, hence for both of them we need new-style implementation to be used when we attempt to compile describe. Otherwise it won't compile in new Numba pipeline.